Dockerfile

```diff
@@ -50,23 +67,32 @@ RUN \
  && pip3 install kafka-python==1.3.2 \
  && pip3 install bokeh==0.12.4
 
+# Create a directory to
+RUN mkdir ~/.ssh 
+COPY ./config/ssh_config .ssh/config
+
 RUN \
-# Sbt
+# git clone during image build and setting enviroment variables
+ cd ~ \
+ && git clone https://yogeshgo05@github.com/abgoswam/streamai
```

Comment:
there is a failure at this step..

<snip>
Step 10/13 : COPY ./config/ssh_config .ssh/config
lstat config/ssh_config: no such file or directory
</snip>